integration_tests/doc_example_driver.sh

```diff
@@ -0,0 +1,142 @@
+
+http -p hbHB --pretty=format --auth $KEEPER_TOKEN: \
+    PATCH http://localhost:5000/products/qserv_distrib \
+    title="Qserv Data Access" \
+    >> ltd_keeper_doc_examples.txt
+
+echo "\n\n---\n" >> ltd_keeper_doc_examples.txt
+
+http -p hbHB --pretty=format --auth $KEEPER_TOKEN: \
+    POST http://localhost:5000/products/lsst_apps/builds/ \
+    slug='b1' \
+    github_requester='jonathansick' \
+    git_refs:='["master"]' \
+    >> ltd_keeper_doc_examples.txt
+
+echo "\n\n---\n" >> ltd_keeper_doc_examples.txt
+
+http -p hbHB --pretty=format --auth $KEEPER_TOKEN: \
+    POST http://localhost:5000/products/lsst_apps/builds/ \
+    slug='b2' \
+    github_requester='jonathansick' \
+    git_refs:='["master"]' \
+    >> ltd_keeper_doc_examples.txt
+
+echo "\n\n---\n" >> ltd_keeper_doc_examples.txt
+
+http -p hbHB --pretty=format --auth $KEEPER_TOKEN: \
+    POST http://localhost:5000/builds/1/uploaded \
+    >> ltd_keeper_doc_examples.txt
+
+echo "\n\n---\n" >> ltd_keeper_doc_examples.txt
+
+http -p hbHB --pretty=format --auth $KEEPER_TOKEN: \
+    GET http://localhost:5000/products/lsst_apps/builds/ \
+    >> ltd_keeper_doc_examples.txt
+
+echo "\n\n---\n" >> ltd_keeper_doc_examples.txt
+
+http -p hbHB --pretty=format --auth $KEEPER_TOKEN: \
+    GET http://localhost:5000/builds/1 \
+    >> ltd_keeper_doc_examples.txt
+
+echo "\n\n---\n" >> ltd_keeper_doc_examples.txt
+
+http -p hbHB --pretty=format --auth $KEEPER_TOKEN: \
+    POST http://localhost:5000/products/lsst_apps/editions/ \
+    slug="latest" \
+    title="Latest" \
+    build_url="http://localhost:5000/builds/1" \
+    tracked_refs:='["master"]' \
+    published_url="pipelines.lsst.io" \
+    >> ltd_keeper_doc_examples.txt
+
+echo "\n\n---\n" >> ltd_keeper_doc_examples.txt
+
+http -p hbHB --pretty=format --auth $KEEPER_TOKEN: \
+    PATCH http://localhost:5000/editions/1 \
+    title="Development master" \
+    >> ltd_keeper_doc_examples.txt
+
+echo "\n\n---\n" >> ltd_keeper_doc_examples.txt
+
+http -p hbHB --pretty=format --auth $KEEPER_TOKEN: \
+    GET http://localhost:5000/editions/1 \
+    >> ltd_keeper_doc_examples.txt
+
+echo "\n\n---\n" >> ltd_keeper_doc_examples.txt
+
+http -p hbHB --pretty=format --auth $KEEPER_TOKEN: \
+    GET http://localhost:5000/products/lsst_apps/editions/ \
+    >> ltd_keeper_doc_examples.txt
+
+echo "\n\n---\n" >> ltd_keeper_doc_examples.txt
+
+http -p hbHB --pretty=format --auth $KEEPER_TOKEN: \
+    POST http://localhost:5000/editions/1/rebuild \
+    build_url="http://localhost:5000/builds/2" \
+    >> ltd_keeper_doc_examples.txt
+
+echo "\n\n---\n" >> ltd_keeper_doc_examples.txt
+
+http -p hbHB --pretty=format --auth $KEEPER_TOKEN: \
+    DELETE http://localhost:5000/builds/1 \
+    >> ltd_keeper_doc_examples.txt
+
+echo "\n\n---\n" >> ltd_keeper_doc_examples.txt
+
+http -p hbHB --pretty=format --auth $KEEPER_TOKEN: \
+    DELETE http://localhost:5000/editions/1 \
+    >> ltd_keeper_doc_examples.txt
```

Comment:
Unless there is a specific reason to put it in ./ltd_keeper_doc_examples.txt, I'd output this to stdout and let me redirect.